.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,31 @@
+**Home Assistant release (`hass --version`):**
+
+
+**Python release (`python --version`):**
```

Comment:
I'm not sure what Python version the majority of distributions are shipping nowadays. At least Fedora 23 has Python 3.4.3 as default.